code/controllers/subsystem.dm

```diff
@@ -1,88 +1,91 @@
 /**
 	* # Subsystem base class
-	* 
+	*
```

Comment:
please turn off strip trailing whitespace setting in your IDE

Reply:
iirc this is actually an automatic one now in our editorconfig and this is just an older file.